src/App.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import './App.scss';
 
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is correctly implemented and exported from './components/Person/Person'. If there are any issues with the import path or the component itself, it may cause rendering problems.